app/assess/routes.py

```diff
@@ -109,13 +108,11 @@ def view_application(application_id, fund_id, round_id):
     if not application_data:
         abort(404)
 
-    status_data_all = get_status_data(
-        APPLICATION_STORE_API_HOST, application_id
-    )
-    status_not_completed = get_status_not_completed(status_data_all)
-    status_completed = get_status_completed(status_data_all)
+    status_data_all = get_statuses(application_id)
     if not status_data_all:
         abort(404)
```

Comment:
Should not retrieving statuses issue a 404? It should probably give an internal service error or a different code.

Reply:
Okay! will remove it